src/renderer/i18n.ts

```diff
@@ -37,11 +37,11 @@ export class LocalizationStore {
     reaction(() => this.activeLang, lang => storage.set(lang));
   }
 
-  async init() {
-    await this.setLocale(this.activeLang);
+  init() {
+    this.setLocale(this.activeLang);
   }
 
-  async setLocale(locale: string) {
+  setLocale(locale: string) {
```

Comment:
I think it's better to keep it `async`.
Currently it works only because all available locales are bundled in the app and no actual loading is happening.

Reply:
Ah oops fair

Reply:
However, the types of the package we are using are all `(...) => void`